src/Handlers/AdditionalDataHandlerInterface.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Tomaj\NetteApi\Handlers;
+
+interface AdditionalDataHandlerInterface
```

Comment:
feel free to suggest interface name

Reply:
not great not terrible ;-D we can go with this

Reply:
So? Will you merge it?🤪